Runtime/Impulse/CinemachineCollisionImpulseSource.cs

```diff
@@ -123,13 +123,13 @@ private void GenerateImpactEvent(Collider other, Vector3 vel)
             // Calculate the signal direction and magnitude
             float mass = GetMassAndVelocity(other, ref vel);
             if (m_ScaleImpactWithSpeed)
-                mass *= vel.magnitude;
-            Vector3 dir = Vector3.down;
+                mass *= Mathf.Sqrt(vel.magnitude);
```

Comment:
Why sqrt here?

Reply:
because it feels better.  This is very seat-of the-pants... not squaring it makes it last too long